### ISSUE TITLE
The website is unusable without JavaScript

### ISSUE THREAD

**Issue by KOLANICH** [colour-science/colour-science.org#2]
Being workable without JavaScript is a mandatory requirement for every website in 2019.

**Comment by thomasmansencal** [comment]
Hi @KOLANICH,

Is it? In any case I don't have the bandwidth to change anything about it at the moment, feel free to open a PR as it seems like the main issue is the navigation breaking. I came across this: https://www.freakspot.net/barra-de-navegacion-de-bootstrap-sin-javascript-en.html

**Comment by KOLANICH** [comment]
Yes, something like that, or maybe even just utilise `<details>` tag for that.

**Comment by thomasmansencal** [comment]
There is a discussion about that here: https://github.com/twbs/bootstrap/discussions/36651
